boards/lilygo/ttgo_lora32/ttgo_lora32_esp32_procpu_defconfig

```diff
@@ -8,3 +8,4 @@ CONFIG_UART_CONSOLE=y
 
 CONFIG_GPIO=y
 CONFIG_I2C=y
+CONFIG_ESP32_USE_UNSUPPORTED_REVISION=y
```

Comment:
Can you add a comment on why this is needed (if this is needed?). This way it's not only clearer to the user but also helps knowing when we might be able to remove it in the future

Reply:
At a minimum, makes this a different (and first) commit, similar to https://github.com/zephyrproject-rtos/zephyr/commit/956edd1c3dbf539f1e4b472c61e78668c5e85a16 since my understanding is that it is an unrelated (but needed) change

Reply:
fwiw i was not suggesting to add a comment to the git commit (although it's not a bad idea to do it too :) ) but rather right in the defconfig file so that people understand where this is coming from. BTW I am not sure this really belongs to the defconfig as this is not something that should be modifiable, no? Shouldn't it be `select`ed in the Kconfig file instead? cc @sylvioalves

Reply:
It says "unsupported revision" - at least my board uses this version of the chip.
Maybe newer boards (it's still in production) use a newer version.
I'm not sure this setting has any impact when selected for a chip revision > 1

 I added a comment and also trimmed the commit log line length.

Reply:
```
config ESP32_USE_UNSUPPORTED_REVISION
	bool "Use unsupported ESP32 revision (Rev 0/1)"
	help
	  ESP32 SoC has multiple revisions, some of which are not supported by the current
	  implementation, as such as REV0 and REV1. In case those revisions are required,
	  set this option to enable support for them. Note that this is not recommended and
	  may lead to unexpected behavior.
```

rev0 and rev1 ESP32 SocS (old versions) have rom bugs that we haven't and will not address. So if the board used such old chip revision, this config must be set.